File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { ManualPrecipitationStationsSDK } from '..'


describe('exists', async () => {

  test('test-mode', () => {
    const testsdk = ManualPrecipitationStationsSDK.test()
    equal(testsdk instanceof ManualPrecipitationStationsSDK, true,
      'ManualPrecipitationStationsSDK.test() must return a client synchronously')
  })

})

File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("manual-precipitation-stations_sdk")

describe("ManualPrecipitationStationsSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
